error: use of undeclared identifier 'INFO_HASH_LENGTH'

I encountered this error when building with the `--disable-bittorrent` flag with the commit 3330110cafebd86b231d860cddea2f36ca8e755c.
> make  all-recursive
make[1]: Entering directory '/home/acer/aria2/aria2'
Making all in po
make[2]: Entering directory '/home/acer/aria2/aria2/po'
make[2]: Leaving directory '/home/acer/aria2/aria2/po'
Making all in lib
make[2]: Entering directory '/home/acer/aria2/aria2/lib'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/acer/aria2/aria2/lib'
Making all in deps
make[2]: Entering directory '/home/acer/aria2/aria2/deps'
make[3]: Entering directory '/home/acer/aria2/aria2/deps'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/home/acer/aria2/aria2/deps'
make[2]: Leaving directory '/home/acer/aria2/aria2/deps'
Making all in src
make[2]: Entering directory '/home/acer/aria2/aria2/src'
Making all in includes
make[3]: Entering directory '/home/acer/aria2/aria2/src/includes'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/acer/aria2/aria2/src/includes'
make[3]: Entering directory '/home/acer/aria2/aria2/src'
  CXX      main.o
  CXX      A2STR.lo
  CXX      AbstractAuthResolver.lo
  CXX      AbstractCommand.lo
  CXX      AbstractDiskWriter.lo
  CXX      AbstractHttpServerResponseCommand.lo
  CXX      AbstractOptionHandler.lo
  CXX      AbstractProxyRequestCommand.lo
  CXX      AbstractProxyResponseCommand.lo
  CXX      AbstractSingleDiskAdaptor.lo
  CXX      AdaptiveFileAllocationIterator.lo
  CXX      AdaptiveURISelector.lo
  CXX      AuthConfig.lo
  CXX      AuthConfigFactory.lo
  CXX      AutoSaveCommand.lo
  CXX      BackupIPv4ConnectCommand.lo
  CXX      base32.lo
  CXX      bitfield.lo
  CXX      BitfieldMan.lo
  CXX      BufferedFile.lo
  CXX      ByteArrayDiskWriter.lo
  CXX      CheckIntegrityCommand.lo
  CXX      CheckIntegrityDispatcherCommand.lo
  CXX      CheckIntegrityEntry.lo
  CXX      Checksum.lo
  CXX      ChecksumCheckIntegrityEntry.lo
  CXX      ChunkChecksum.lo
  CXX      ChunkedDecodingStreamFilter.lo
  CXX      ColorizedStream.lo
  CXX      Command.lo
  CXX      ConnectCommand.lo
  CXX      console.lo
  CXX      ConsoleStatCalc.lo
  CXX      ContentTypeRequestGroupCriteria.lo
  CXX      Context.lo
  CXX      ContextAttribute.lo
  CXX      Cookie.lo
  CXX      CookieStorage.lo
  CXX      cookie_helper.lo
  CXX      CreateRequestCommand.lo
  CXX      CUIDCounter.lo
  CXX      DefaultAuthResolver.lo
  CXX      DefaultBtProgressInfoFile.lo
  CXX      DefaultDiskWriter.lo
  CXX      DefaultDiskWriterFactory.lo
  CXX      DefaultPieceStorage.lo
DefaultBtProgressInfoFile.cc:267:24: error: use of undeclared identifier 'INFO_HASH_LENGTH'
  267 |   if (infoHashLength > INFO_HASH_LENGTH ||
      |                        ^
DefaultBtProgressInfoFile.cc:268:26: error: use of undeclared identifier 'INFO_HASH_LENGTH'
  268 |       (infoHashLength != INFO_HASH_LENGTH && infoHashCheckEnabled)) {
      |                          ^
DefaultBtProgressInfoFile.cc:272:31: error: use of undeclared identifier 'INFO_HASH_LENGTH'
  272 |     std::array<unsigned char, INFO_HASH_LENGTH> savedInfoHash;
      |                               ^
3 errors generated.
make[3]: *** [Makefile:2638: DefaultBtProgressInfoFile.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/home/acer/aria2/aria2/src'
make[2]: *** [Makefile:2679: all-recursive] Error 1
make[2]: Leaving directory '/home/acer/aria2/aria2/src'
make[1]: *** [Makefile:557: all-recursive] Error 1
make[1]: Leaving directory '/home/acer/aria2/aria2'
make: *** [Makefile:468: all] Error 2

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -9,9 +9,12 @@ jobs:
         os: [ubuntu-22.04, macos-11]
         compiler: [gcc, clang]
         crypto: [openssl, gnutls]
+        bittorrent: [with-bt, without-bt]
         exclude:
         - os: macos-11
           crypto: gnutls
+        - crypto: openssl
+          bittorrent: without-bt
 
     runs-on: ${{ matrix.os }}
 
@@ -73,14 +76,20 @@ jobs:
 
         echo 'CPPFLAGS='"$CPPFLAGS" >> $GITHUB_ENV
         echo 'LDFLAGS='"$LDFLAGS" >> $GITHUB_ENV
+    - name: Disable BitTorrent
+      if: matrix.bittorrent == 'without-bt'
+      run: |
+        FEATURE_FLAGS="$FEATURE_FLAGS --disable-bittorrent"
+
+        echo 'FEATURE_FLAGS='"$FEATURE_FLAGS" >> $GITHUB_ENV
     - name: Configure autotools (Linux, gnutls)
       if: runner.os == 'Linux' && matrix.crypto == 'gnutls'
       run: |
-        ./configure --with-gnutls --without-openssl
+        ./configure --with-gnutls --without-openssl $FEATURE_FLAGS
     - name: Configure autotools (Linux, openssl)
       if: runner.os == 'Linux' && matrix.crypto == 'openssl'
       run: |
-        ./configure --without-gnutls --with-openssl
+        ./configure --without-gnutls --with-openssl $FEATURE_FLAGS
     - name: Configure autotools (macOS)
       if: runner.os == 'macOS'
       run: |

--- a/src/DefaultBtProgressInfoFile.cc
+++ b/src/DefaultBtProgressInfoFile.cc
@@ -57,6 +57,7 @@
 #include "DownloadContext.h"
 #include "BufferedFile.h"
 #include "SHA1IOFile.h"
+#include "BtConstants.h"
 #ifdef ENABLE_BITTORRENT
 #  include "PeerStorage.h"
 #  include "BtRuntime.h"

